doc/lsst.ap.verify/running.rst

```diff
@@ -30,6 +30,15 @@ Using the `Cosmos PDR2`_ CI dataset as an example, first setup the dataset, if i
 
 You will need to setup the dataset once each session.
 
+Next, download and setup the model packages required to run the Real-Bogus (RB) Classifier as follows.
+
+.. prompt:: bash
+
+   git clone https://github.com/lsst-dm/rbClassifier_data
+   setup -r /my/path/rbClassifier_data
```

Comment:
Use the same code format as the Cosmos PDR2 example above, since the user needs to take the same actions.